routes/buzz-words.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const saveBuzzWords = require('../save-buzz-words');
+const express = require('express');
+const router = express.Router();
+const bodyParser = require('body-parser');
```

Comment:
This line and the next line should be in your app.js file. Then it'll apply to both your route files.